src/app/(sidebar)/smart-contracts/contract-explorer/components/ContractInfo.tsx

```diff
@@ -264,7 +274,7 @@ export const ContractInfo = ({
       },
     };
 
-    const badge = type ? item[type] : item.unverified;
+    const badge = hasWasmData ? item.verified : item.unverified;
```

Comment:
nice!

Reply:
That's correct. If no data is returned from that call, it means we couldn't fetch the repo data, or the validation checks failed.